src/klassy.js

```diff
@@ -0,0 +1,12 @@
+import bem from 'react-bem-helper';
+
+export default function klassy (packageJSON, override) {
```

Comment:
This function is used as a convention enforcer: 

1. every component should be a npm module with `package.json`
2. component name === package name === BEM block